### DISCUSSION TITLE
Linodave with Excel 64 bits

### DISCUSSION THREAD

**Discussion by zigouiik** [dotnetprojects/DotNetSiemensPLCToolBoxLibrary#200] [Q&A]
Hello all,
I've built a project based on libnodave on Excel 32bits. 
It's working fine untill my firm changed to Office 365 64 bits.
Now I've modify the call of external sub and functions with writing 'ptrsafe' in the declaration.
Problem is that it seems not working with the original DLL.
I've got an error'48' saying that the DLL can't be found.
Is there someone who has faced this issue and can help me to solve it ?
I've tried the DLLs from the library without success.
Thanks in advance

**Comment by jogibear9988** [comment]
You need a 64Bit libnodave
My library here includes a 32 bit and a 64 bit version of a modified libnodave

**Comment by zigouiik** [comment]
Hello and thanks for your reply jogibear9988,

I've tried to use your DLLs and that didn't work for me.
I'm a bit confused by the way.., 
I have modify another 32 bits Excel project to fit my 64 bits version and it's OK, but this one didn't use any DLL. 
So I'm still thinking that for my libnodave Excel project it's a DLL problem.

I've add a Function (Public Declare PtrSafe Function SetCurrentDirectory Lib "kernel32" Alias "SetCurrentDirectoryA" (ByVal lpPathName As String) As Long) to use the actual folder to found the DLL (working in 32 bits version) , but I also put the DLL in the System32 folder and in the Syswow64 folder.
The declarations seems to be correct but it keeps telling me "error 48" when I try to connect to a PLC. 
It's the first DLL function I use when I open the workbook.

Well I know it's a dot net section here but maybe someone has even faced this issue and know what to do...

Thanks a lot for your replies
